client/components/blank-canvas/index.tsx

```diff
@@ -38,10 +52,22 @@ BlankCanvas.Header = ( { backUrl, children, onBackClick } ) => (
 				{ __( 'Back' ) }
 			</Button>
 		) }
-		{ children.length && <div className="blank-canvas__header-title">{ children }</div> }
+		{ children && <div className="blank-canvas__header-title">{ children }</div> }
 	</div>
 );
-BlankCanvas.Content = ( { children } ) => <div className="blank-canvas__content">{ children }</div>;
-BlankCanvas.Footer = ( { children } ) => <div className="blank-canvas__footer">{ children }</div>;
+
+interface ContentProps {
+	children: ReactNode;
+}
+BlankCanvas.Content = ( { children }: ContentProps ) => (
+	<div className="blank-canvas__content">{ children }</div>
+);
+
+interface FooterProps {
+	children: ReactNode;
+}
```

Comment:
It seems that content and footer props have the same type.
No a blocker at all, just thinking out loud. Is the `PropsWithChildren< {} >` an option?

Reply:
I can use `PropsWithChildren` alone; if I add the {}, the lining will complain because of an empty prop element.